src/components/pages/auth/SignUpForm.jsx

```diff
@@ -188,10 +193,12 @@ const SignUpForm = () => {
             checked={formik.values.accept_policy}
             onChange={formik.handleChange}
           />
-          <label className="cursor-pointer select-none" htmlFor="accept_policy">
+          <label className="cursor-pointer select-none mr-2" htmlFor="accept_policy">
             {t('iAgreeWith')}
           </label>
-          <a className="text-theme-1 ml-1" href="/docs/PrivacyPolicyEn.html" target="_blank">{t('privacyPolicy')}</a>.
+          <a className="text-theme-1" href={` ${process.env.PUBLIC_URL}${policy}`} target="_blank">
+            {t('privacyPolicy')}
+          </a>
```

Comment:
you missed dot
```suggestion
          </a>.
```

Reply:
i delete it purposely,  I thought that is mistake ))))